ui/pages/notifications/notification-components/erc1155-sent-received/erc1155-sent-received.tsx

```diff
@@ -189,7 +192,7 @@ export const components: NotificationComponent<ERC1155Notification> = {
     },
   },
   footer: {
-    type: 'footer_onchain_notification',
+    type: NotificationComponentType.OnChainFooter,
```

Comment:
Really appreciate this abstraction! I really like the exhaustive discrimination we can now do for OnChain, Feature Announcement, and now Snap Body/Footer/Etc.